### ISSUE TITLE
near constant 100% CPU use on macOS with VS Code

### ISSUE THREAD

**Issue by witt3rd** [elba/idirs-elba-interface#2]
Using the wrapper from VS Code on macOS and the Activity Monitor shows `idris-elba-interface` at the top by CPU utilization where it is warming the room at near 100% sustained.  This is without doing anything in the IDE - having just launched it and opened an `.idr` file.  It also does not stop after quitting VS Code.

**Comment by andylokandy** [comment]
Oh, were you using Idris2? Since this crate has been out of date for a long time, it may not be compatible with the latest compiler.

**Comment by witt3rd** [comment]
No, just Idris.  I commented out the file watching code and it stopped.
